src/ResourceManager/Storage/ChangeLog.md

```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Current Release
+* Update the parameters so that -ResourceGroupName parameter is required when using -Name parameter.
```

Comment:
@AMoghrabi would you mind moving this snippet from the `Storage` change log to the `TrafficManager` change log?